x-pack/plugins/ml/public/application/components/navigation_menu/main_tabs.tsx

```diff
@@ -154,40 +154,25 @@ export const MainTabs: FC<Props> = ({ tabId, disableLinks }) => {
   }, [selectedTabId]);
 
   return (
-    <EuiTabs display="condensed">
-      {tabs.map((tab: Tab) => {
+    <EuiPageHeader
+      paddingSize="m"
+      bottomBorder
+      tabs={tabs.map((tab: Tab) => {
```

Comment:
@peteharverson I updated ML's tab navigation to pass the tab props through to a EuiPageHeader component. This fixes the display of them and will better align with other apps as well.

Ex:
<img width="988" alt="Screen Shot 2021-06-24 at 15 32 30 PM" src="https://user-images.githubusercontent.com/549577/123321801-6fa99380-d501-11eb-8d38-863fda758ac6.png">

<img width="1349" alt="Screen Shot 2021-06-24 at 15 02 54 PM" src="https://user-images.githubusercontent.com/549577/123320856-4d634600-d500-11eb-9c4d-aa2a5fb7d532.png">

Reply:
The switch to `EuiPageHeader` gives `16px` padding around the tabs - just wondering if this is giving the tab navigation too much height? See below for comparison to the previous styling?

<img width="1165" alt="nav_menus_713" src="https://user-images.githubusercontent.com/7405507/123617433-23ab6700-d7ff-11eb-8fe2-9510dcb13d5f.PNG">

Reply:
Unclear what @cchaos wanted for this one. I've seen it both ways. I've applied a small patch to bring it closer to the border. Since it's likely off hours for you, I'll assume this is good (since we have all reviews besides ML done) I'll merge, and then adjust more if you need.

![image](https://user-images.githubusercontent.com/324519/123825552-fe852a00-d8cc-11eb-825f-8374e1e3dce0.png)

Reply:
Dropping the `padding-bottom` to `0` looks better thanks @snide. 7.14 / 7.13 comparison below. New styling uses slightly more vertical space, but I think it looks ok given the slightly increased font size.

<img width="1309" alt="nav_menus_comparison" src="https://user-images.githubusercontent.com/7405507/123828508-7701e000-d8f9-11eb-9af8-ee80acf957bc.PNG">